code/modules/mob/living/silicon/ai/freelook/chunk.dm

```diff
@@ -1,34 +1,49 @@
-#define UPDATE_BUFFER 25 // 2.5 seconds
+#define UPDATE_BUFFER_TIME 25
```

Comment:
why yeet the comment? also should this be `(2.5 SECONDS)`?

Reply:
now defined properly